packages/services/schema/src/orchestrators.ts

```diff
@@ -273,6 +348,7 @@ const createFederation: (
               filter,
             );
             return {
+              // @note Although it can differ from the supergraph's, ignore metadata on contracts.
               ...subgraph,
               typeDefs: filteredSubgraph.typeDefs,
```

Comment:
What is the reason here for ignoring the metadata?

Reply:
Metadata will only differ between contracts and supergraph for `@shared` fields or entities, where one subgraph could impact another.
I think this is an edge case and the additional complexity of calculating and displaying metadata doesn't provide much benefit to the users.